python/tests/io/test_hdf5.py

```diff
@@ -7,11 +7,15 @@
 import tempfile
 
 
-def check_roundtrip(obj):
+def roundtrip(obj):
```

Comment:
How about an in-memory file using `BytesIO`? `h5py` can handle those.

Reply:
Could be a good suggestion for the future, but unrelated to this change, so I suggest not doing this now.